### DISCUSSION TITLE
Mark or color groups differently depending on whether they're added in scene or in instance

### DISCUSSION THREAD

**Discussion by Zireael07** [godotengine/godot-proposals#5904] [Editor]
Related to https://github.com/godotengine/godot/issues/69649

(Opening here because I don't have a mockup nor code)

Personally I have been mixing and matching groups added in scene and on instance, which means I would've ran into the issue reported myself at some point.

Please either color-code them differently (Duriel suggests graying some of them out) and/or add an icon.

**Comment by Mickeon** [comment]
I think global groups would HEAVILY addresses this issue without necessarily needing more style to each group: https://github.com/godotengine/godot/pull/60965.

Doesn't mean they can't be later implemented in addition to them.
- See also https://github.com/godotengine/godot-proposals/issues/5846.
